packages/manager/src/hooks/usePlacementGroupsData.ts

```diff
@@ -31,9 +31,7 @@ export const usePlacementGroupData = ({
   const { data: linodes, error, isLoading } = useAllLinodesQuery(
     {},
     {
-      '+or': placementGroup?.linodes.map(({ linode }) => ({
-        linode,
-      })),
+      '+or': placementGroup?.linodes.filter(({ linode }) => linode),
```

Comment:
I think this is the filter we want here

```suggestion
      '+or': placementGroup?.linodes.map((linodeItem) => ({
        id: linodeItem.linode,
      })),
```

Reply:
yup that seems about right - not for later - optimize queries when working with real data